version.txt

```diff
@@ -1 +1,2 @@
-0.86.0-SNAPSHOT
+0.85.2-SNAPSHOT
```

Comment:
If the relase plugin takes care of ther version, can we fill up this the blank? I think `0.85.2-SNAPSHOT` is little confusing.